scalafix/rules/src/main/scala/fix/Scalacollectioncompat_NewCollections.scala

```diff
@@ -30,6 +30,28 @@ case class Scalacollectioncompat_NewCollections(index: SemanticdbIndex)
     Symbol("_root_.scala.runtime.Tuple3Zipped.Ops.zipped.")
   )
 
+  val retain = 
+    SymbolMatcher.normalized(
+      Symbol("_root_.scala.collection.mutable.MapLike.retain.")
```

Comment:
Oops, I’ve just noticed that this operation is also on `mutable.Set`. We should support that use case too…